### PR TITLE
Notify about release branch update MAPSMBL-152

### PR DESCRIPTION
**Pull request by yunikkk**
### Summary of changes

- Added notifications about release branch updates
- Changed python script to send single relevant parameter with the merged branch name
 
## Pull request checklist:
 - [x] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Run `make update-api` to update generated api files, if there's public API changes, otherwise the `verify-api-*` CI steps might fail.
 - [x] Update [CHANGELOG.md](../CHANGELOG.md) or use the label 'skip changelog', otherwise `check changelog` CI step will fail.
 - [ ] If this PR is a `v10.[version]` release branch fix / enhancement, merge it to `main` firstly and then port to `v10.[version]` release branch.

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

File: scripts/ci-circleci-start-pipeline.py [hunk 1]
```diff
@@ -66,13 +67,19 @@ def Main():
...
-        "mapbox_android_upstream": True,
-        "mapbox_slug": args.origin_slug,
-        "mapbox_android_hash": args.hash,
-        "mapbox_merge_main": args.current_branch == MAIN_BRANCH_NAME,
-        "benchmark_date": created
-    }
+    if args.sync_internal:
+        params = {
+            "mapbox_slug": args.origin_slug,
+            "mapbox_sync_branch": args.current_branch
```
Comment (yunikkk) [inline scripts/ci-circleci-start-pipeline.py:73]: Only send relevant parameters to avoid declaring them in the upstream Circle config

File: scripts/ci-circleci-start-pipeline.py [hunk 2]
```diff
@@ -66,13 +67,19 @@ def Main():
...
+        }
+    else:
+        params = {
+            "mapbox_android_upstream": True,
+            "mapbox_slug": args.origin_slug,
```
Comment (alexander-kulikovskii) [inline scripts/ci-circleci-start-pipeline.py:78]: nit:
better to add `mapbox_slug` after `if` block:

params[mapbox_slug] = args.origin_slug

File: plugin-overlay/gradle.properties [hunk 3]
```diff
@@ -2,5 +2,4 @@ POM_GROUP_ID=com.mapbox.plugin
 POM_ARTIFACT_ID=maps-overlay
 POM_ARTIFACT_TITLE=The map overlay module for the Mapbox Maps SDK for Android
 POM_DESCRIPTION=The map overlay module for the Mapbox Maps SDK for Android
-BINTRAY_PACKAGE_NAME=POM_GROUP_ID + ":" + POM_ARTIFACT_ID
```
Comment (alexander-kulikovskii) [inline plugin-overlay/gradle.properties:5]: why did you remove it here? not for all modules?

Comment (yunikkk) [reply plugin-overlay/gradle.properties:5]: It's a leftover that's not present anywhere for a long time after we stopped publishing to Bintray

Comment (yunikkk) [reply plugin-overlay/gradle.properties:5]: And these 3 places were the only ones where this constant was still declared

Comment (pengdev) [reply plugin-overlay/gradle.properties:5]: yeah, we can safely remove these fields as we are not publishing to bintray any more

**Review by pengdev — APPROVED** [review]

**Review by ank27 — APPROVED** [review]

**Review by alexander-kulikovskii — APPROVED** [review]

**Review by kiryldz — APPROVED** [review]
